tasks/image-upscale/__init__.py

```
@@ -0,0 +1,71 @@
+    
+    # Step 1: Start upscale task
+    start_url = "https://console.oomol.com/api/tasks/fal/images/upscale/start"
+    start_payload = {"image_url": image_url}
+
+    api_key: Any = context.oomol_llm_env.get("api_key")
+    headers = {
+        'Authorization': f"Bearer {api_key}",
+    }
+    
+    try:
+        start_response = requests.post(start_url, json=start_payload,headers=headers)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)